publishing/datasetAPI/put_version_test.go

```diff
@@ -436,15 +440,75 @@ func TestFailureToUpdateVersion(t *testing.T) {
 		}
 	})
 
+	// test for updating a version so it with a state of `published` but missing mandatory fields
+	Convey("Given a published dataset and edition and an unpublished version exist ...", t, func() {
```

Comment:
dont need `...` in the convey statements throughout this test. also `so it with a state` does not make sense in the comment